File: Submissions/ABC/ABC030/B/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);

    int n = sc.nextInt();
    int m = sc.nextInt();

    sc.close();

    if (12 <= n) {
      n = n - 12;
    }

    double H = n * 30 + m * 0.5;
    double M = m * 6;

    double res = Math.abs(H - M);

    if (180 < res) {
      res = 360 - res;
    }
    
    System.out.println(res);

  }

}

File: Submissions/ABC/ABC030/B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  double n, m;
  cin >> n >> m;

  if (12 <= n) {
    n = n - 12;
  }

  n = n * 30 + m * 0.5;
  m = m * 6;

  double res = abs(n - m);
  
  if (180 < res) {
    res = 360 - res;
  }

  cout << res << endl;

}